react-blocks/src/blocks/Counter/CounterIcon.js

```diff
@@ -0,0 +1,7 @@
+import { React, Component } from 'react';
```

Comment:
you can use a different icon here so that don't end up with the same icons
or it could be a dashicon
https://developer.wordpress.org/block-editor/developers/block-api/block-registration/#icon-optional